src/javascript/app/pages/user/metatrader/metatrader.config.js

```diff
@@ -126,56 +126,83 @@ const MetaTraderConfig = (() => {
                 resolve(needsRealMessage());
             } else {
                 BinarySocket.wait('get_settings').then(() => {
-                    const response_get_settings = State.getResponse('get_settings');
-
-                    const showCitizenshipMessage = () => {
-                        $message.find('.citizen').setVisibility(1).find('a').attr('onclick', `localStorage.setItem('personal_details_redirect', '${acc_type}')`);
+                    const showCitizenshipMessage = () => $message
```

Comment:
please move $message to a new line, then indent everything after that by one more

Reply:
and we don't need to return anything, so put `{}` around it